src/query/server/server.go

```diff
@@ -191,7 +191,7 @@ func Run(runOpts RunOptions) {
 		defer cleanup()
 	}
 
-	engine := executor.NewEngine(backendStorage)
+	engine := executor.NewEngine(backendStorage, scope.SubScope("engine"))
```

Comment:
nit: Maybe use `query_engine` for additional context?

Reply:
already shows up as coordinator_engine.